lib/money/src/Money.cpp

```diff
@@ -0,0 +1,85 @@
+#include "Money.h"
+
+Money::Money(double amount, Currency currency)
+	: USCAmount(converter.ConvertToUSC(amount, currency))
+	, currency(currency)
+{}
+
+Money Money::operator+=(const Money other) {
+	this->USCAmount += other.USCAmount;
+	return *this;
+}
+
+Money Money::operator-=(const Money other) {
+	this->USCAmount -= other.USCAmount;
+	return *this;
+}
+
+std::ostream& operator<<(std::ostream& os, const Money other) {
+	return other.Print(os);
+}
+
+std::istream& operator>>(std::istream& is, Money& other) {
+	std::cout << "Input currency (USD, DKK): ";
```

Comment:
Don't mix interaction with a user and reading from the input stream.
The input stream can be a file or a string stream.